.github/workflows/helm-chart-releaser.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           app_version: ${{ steps.vars.outputs.tag }}
   
       - name: Manually publish Helm chart with release version
-        if: ${{github.event.inputs.release-ver}} != "v"
+        if:  github.event_name != 'pull_request' && ${{github.event.inputs.release-ver}} != "v"
```

Comment:
@CyrineG or @Revolyssup will you add commentary as to the need for this change?

Reply:
I did this because publishing helm charts on a pull request doesn't make sense and requires permissions which fails the workflows. Also idk why but this didnt take effect as the release-workflow on PR basis is still being executed.

Reply:
Sound logic. Thanks for this explanation. 👍